manifest.json

```diff
@@ -22,9 +22,7 @@
       "react-native@0.48.3",
       "react-native-electrode-bridge@1.5.10"
     ],
-    "targetJsDependencies": [
-      "react@16.0.0-alpha.12"
-    ]
+    "targetJsDependencies": ["react@16.0.0-alpha.12"]
```

Comment:
Sorry for the mess, don't know how to disable prettier in this file. The only change is `flash-list` being added.